test/compiler_test.exs

```diff
@@ -2338,6 +2338,40 @@ defmodule CompilerTest do
       assert {init_fn, predict_fn} = Axon.compile(mp_model)
       assert Nx.type(predict_fn.(init_fn.(), Nx.random_uniform({1, 1, 3, 3}))) == {:bf, 16}
     end
+
+    test "bilinear without aligned corners" do
```

Comment:
These tests should be in layer_test.exs. The compiler tests test the Axon model to JIT compilation API

Reply:
Done.